target/linux/ipq807x/patches-5.15/500-rax120-add-g761-driver.patch

```diff
@@ -0,0 +1,34 @@
+Index: linux-5.15.34/drivers/hwmon/Kconfig
+===================================================================
```

Comment:
All patches must be generated using git format-patch with proper name, description, and SoB.